R/infillOptCMAES.R

```diff
@@ -38,27 +38,21 @@ infillOptCMAES = function(infill.crit, models, control, par.set, opt.path, desig
     vectorized = TRUE
   )
 
-  if (!("stop.ons" %in% names(cmaes.control))) {
-    cmaes.control = BBmisc::insert(cmaes.control, list(stop.ons = c(
-      cmaesr::getDefaultStoppingConditions()
-    )))
-  }
+  cmaes.control = insert(list(stop.ons = c(cmaesr::getDefaultStoppingConditions())), cmaes.control)
 
-  if (!("restart.triggers" %in% names(cmaes.control))) {
-    cmaes.control = BBmisc::insert(cmaes.control, list(
-      restart.triggers = BBmisc::extractSubList(cmaes.control$stop.ons, "code")
-    ))
-  }
+  cmaes.control = insert(list(
+      restart.triggers = extractSubList(cmaesr::getDefaultStoppingConditions(), "name")), cmaes.control)
```

Comment:
Better insert cmaes.control$stop.ons instead of getDefaultStoppingConditions

Reply:
I think you are reviewing an old version? I already fixed that. Was a merging error on my side.

Reply:
Am I? Oh, I am :D 
Haven't done it in a long time ...
Looks fine for me now.